pspla/PSPBUILD

```diff
@@ -21,7 +21,7 @@ package() {
     mkdir -m 755 -p "$pkgdir/psp/lib"
     install -m 644 libpspla.a "$pkgdir/psp/lib"
     mkdir -m 755 -p "$pkgdir/psp/include"
-    cp -R pspla.h "$pkgdir/psp/include"
+    cp -R pspla2.h "$pkgdir/psp/include"
```

Comment:
Are there any applications that rely on version 1 that will no longer work now? It seems you could have both installed at the same time, but I'm not sure if that would have any value.

Reply:
Given the fast pace of pspla development, even if any application used version 1 API, they should be switching to version 2 now, so there's no need to keep the old version.